src/OpenFga.Sdk/Api/OpenFgaApi.cs

```diff
@@ -446,6 +447,9 @@ public async Task<ReadAuthorizationModelResponse> ReadAuthorizationModel(string
         if (continuationToken != null) {
             queryParams.Add("continuation_token", continuationToken.ToString());
         }
+        if (startTime != null) {
+            queryParams.Add("start_time", startTime.ToString());
```

Comment:
The default `ToString` looks to format like `2/17/2025 11:44:02 AM`, if we want to get ISO 8601 like the API needs it looks like [we can use](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier) `ToString("O")`

Reply:
Updated to resolve this.  Thanks!